manifests/q/quick-lint/quick-lint-js/2.4.1/quick-lint.quick-lint-js.installer.yaml

```
@@ -0,0 +1,22 @@
+# To the extent possible under law, Matthew "strager" Glazar
+# has waived all copyright and related or neighboring rights
+# to _quick-lint-js winget manifests_. This work is
+# published from United States.
+# http://creativecommons.org/publicdomain/zero/1.0/
+
+PackageIdentifier: quick-lint.quick-lint-js
+PackageVersion: 2.4.1
+
+Installers:
+- InstallerUrl: https://c.quick-lint-js.com/releases/2.4.1/windows/quick-lint-js.msix
+  InstallerSha256: 8601c5a6151991af730994bd048a59b8fe2e0fbb59fa9fb1eb67a4be6b41deb0
+  InstallerType: msix
+  Platform:
+  - Windows.Desktop
+  - Windows.Universal
+  MinimumOSVersion: 10.0.0.0
+  Architecture: x64
+  PackageFamilyName: fa6b940b-8f96-4b46-8e21-703a63133e06_y724facbts4te
```

```suggestion
  PackageFamilyName: fa6b940b-8f96-4b46-8e21-703a63133e06_116557qebcmn6
```
to match Control Panel.

---

Where did you get this code from?

---

I installed the MSIX and ran `$((Get-AppxPackage | Select-Object -Last 1).PackageFamilyName)` in PowerShell.

---

I installed the MSIX and ran `$((Get-AppxPackage | Select-Object -Last 1).PackageFamilyName)` in Windows PowerShell.

---

As a note, the package family name is a combination of the package name and a hash of the publisher name, you can see both in your AppxManifest.xml. The publisher name matches the subject of the signing certificate, so you may have had to change it to sign it with a trusted certificate.

If you are creating the package for the first time, you may want to use a more descriptive package name.

---

Thanks for the advice. I changed the name of the package.